src/locales/pl/translation.json

```diff
@@ -1,145 +1,255 @@
+  "refreshBalance": "Odśwież Saldo",
   "remaining": "Pozostały",
-  "remove": "Usunąć",
-  "removeBoth": "Zarówno usunąć",
+  "remove": "Usuń",
+  "removeBoth": "Usuń",
   "removeLiq": "Usuń Liq",
-  "removeSingle": "usuwać pojedyncze",
-  "reserve": "rezerwować",
+  "removeSingle": "Usuń pojedyńczo",
+  "removeVote": "Usuń Głos",
+  "requiresMoreSupport": "Potrzebuje więcej wsparcia",
+  "reserve": "Rezerwuj",
+  "revenue": "Zysk",
+  "revenueInfo": "Suma opłat Swap i zysku z dywident wygenerowana przez tą pulę w ostatnich {{days}} dniach", 
   "rewards": "Nagrody",
+  "rightNow": "Teraz",
   "router": "Router",
-  "searchAssets": "Szukaj Aktywa ...",
-  "selectAnAsset": "Wybierz atutem",
-  "sell": "Sprzedać",
-  "share": "Dzielić",
-  "shareLink": "Dzielić",
-  "shareViaTwiter": "Podziel się za pośrednictwem Twittera",
-  "slipFeeInfo": "Opłata poślizg jest wstrzykiwany do basenu z tym TXn nagradzać dostawców płynności.",
+  "searchAssets": "Szukaj aktywów...",
+  "search": "Szukaj",
+  "searchPools": "Szukaj pul",
+  "seconds": "sekundy",
+  "selectAnAsset": "Wybierz akrywa",
+  "sell": "Sprzedaj",
+  "shareLink": "Udostępnij",
+  "shareViaTwiter": "Udostępnij na Twitterze",
+  "showHidden": "Pokaż Ukryte",
+  "slip": "Slip",
+  "slipInfo": "Róźnica pomiędzy wartością spot i szacunkową wartością wyjściową",
+  "slipFeeInfo": "The slip fee being injected into the pool with this txn to reward liquidity providers.",
+  "slipTooHigh": "Za Wysoki Slip",
   "sparta": "Sparta",
   "spotPrice": "Cena spot",
-  "stake": "Stawka",
-  "staked": "postawione",
-  "supply": "Dostawa",
-  "swap": "Zamiana",
-  "swapRevenue": "Przychody generowane przez opłata wymiany tej puli w ciągu ostatnich dni {{}} dni",
+  "stake": "Stakuj",
+  "staked": "Zastakowane",
+  "supply": "Podaż",
+  "swap": "Swap",
+  "swapRevenue": "Przychody generowane przez Swap dla tej puli w ciągu ostatnich dni {{}} dni",
+  "swapRevInfo": "Suma nagród przyznanych dostawcom płynności z tej transakcji. Zawiera dywidendy i opłaty slip",
   "swapSpartanProtocol": "Zamień na Spartan protokołu",
-  "synthFactory": "Synth fabryczne",
-  "synths": "syntezatory",
-  "synthVault": "Synth Vault",
-  "synthVaultDetails": "Synth Vault Szczegóły",
-  "tokens": "Żetony",
-  "totalSupply": "całkowitej podaży",
+  "swapVolume": "Wolumen Swap",
+  "swapVolumeInUsd": "Wolumen Swap w $USD",
+  "synthFactory": "Fabryka Synth",
+  "synths": "Synths",
+  "synthVault": "Krypta Synth",
+  "synthVaultDetails": "Szczegóły Krypty Synth",
+  "synthView": "Widok Synth",
+  "synthViewInfo": "Zmienia widok by pokazać szczegóły zasobów synth (wartości APY, synthCaps itp.)",
+  "tokens": "Tokeny",
+  "tokenPrice": "Wartość Tokenu",
+  "totalSupply": "Całkowita podaż",
+  "totalSupplyInfo": "Maksymalna podaż SPARTA to 300M. Jednakże zostało to zaprogramowane w taki sposób, żeby było to niemożliwe do osiągnięcia. Dotarcie blisko 300M zajmie wielę lat nawet bez deflacyjnego feeBurn (palenia opłat)",
```

Comment:
I have updated the English version of this to: "The max supply of SPARTA is 300M. However, the large token burn phase reduced this below 250M which will take 10+ years to come within reach of." so it may need another translation now sorry!

Reply:
No problem, there are few things left to do there anyway. DAO, Positions, Donate and Partners sections were not translated yet. I'll fix this one when I do the rest of the translation